express/src/routes/sign-in.ts

```diff
@@ -40,6 +42,17 @@ router
         processSignInForm("sign-in/enter-password.njk")
     );
 
+router
+    .route("/enter-password-again")
```

Comment:
```suggestion
    .route("/resend-text-code/enter-password")
```
This page is a part of the resend code flow, so let's reflect it in the URL